ffi/dotnet/NowProto/README.md

```diff
@@ -0,0 +1,10 @@
+NowProto client protocol library (C#)
+=====================================
+
+This library provides `NowProto` prototocol implementation (C#) for client
+side `devolutions-agent` DVC implementation.
+
+- Protocol specification: [ironrdp/NowProto](https://github.com/Devolutions/IronRDP/blob/master/docs/NOW-spec.md)
+
+### Notes:
+- Types, which are defined by `NowProto`, but currently unused (e.g. `NowLrgStr`) are not implemented (will be removed from protocol specification in future).
```

Comment:
NOTE: I plan to push PR with NowProto protocol improvements/clean-up tomorrow, the planned scope of changes:
- Remove unused types and constants.
- Adjust error/status reporting messages.
- Clarify the flow of execution session messages between client and server.

(Do not auto-merge this PR yet)